server/@types/bapv.d.ts

```diff
@@ -65,6 +65,7 @@ export type VisitorListItem = {
   address?: string
   restrictions: Restriction[]
   selected?: boolean
+  banned: boolean
```

Comment:
Added this attribute at the 'top' level to avoid having to search through `restrictions` for `BAN` in multiple places.